config/manufacturer_specific.xml

```diff
@@ -63,6 +64,10 @@
 		<Product type="0003" id="004b" name="Smart Energy Switch v2" config="aeon_labs/ses2.xml"/>
 		<Product type="0004" id="0025" name="Z-Wave Repeater"/>
 		<Product type="0004" id="0050" name="DSD31 Siren Gen5" config="aeon_labs/dsd31.xml"/>
+        <!-- US-market products start with type prefix "01" -->
```

Comment:
thats fine. I might drop Aeon Labs a note and ask to get all their product codes etc.